src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.ts

```diff
@@ -1,19 +1,65 @@
 import { IDAO } from "entities/DealRegistrationTokenSwap";
 import { availableSocialMedias } from "../../dealWizardTypes";
-import { bindable, inject } from "aurelia";
+import { bindable, inject, observable } from "aurelia";
 import { IValidationController } from "@aurelia/validation-html";
+import { FirestoreService } from "services/FirestoreService";
+import { IDAOsData } from "services";
+import { ethers } from "ethers";
+import { EnsService } from "services";
+import { Utils } from "services/utils";
+import { AutoCompleteSelect } from "cl-webcomp-poc/components/AutocompleteSelect";
+
+interface IAutocompleteSelectItem {
+  name: string,
+  avatarUrl?: string,
+  id: string,
+  treasury?: string[],
+}
+
+interface IAutocompleteSelectEvent extends Event {
+  detail: IAutocompleteSelectItem
+}
+
+interface ITreasuryAddressItem {
+  name: string,
+  value: string
+}
+
+enum AutocompleteSelectEvent {
+  SelectionChanged = "daoSelectionChanged",
+  NewAdded = "newDaoAdded",
+  Cleared = "inputCleared",
+}
+
+const LAST_1_DAY = Date.now() - (24 * 60 * 60 * 1000)
+const DAO_PLACEHOLDER_AVATAR = "DAO_placeholder.svg";
 
 @inject()
 export class DaoStageContent {
   @bindable name: string;
-  @bindable disabled = false;
+  @bindable disabled: boolean = false;
   @bindable data: IDAO;
   @bindable dataMata: number;
   @bindable someTest: number;
 
+  private daosData: Record<string, IDAOsData>;
+  private daosList: IAutocompleteSelectItem[] = [];
+  private daoListStr: string = "[]";
```

Comment:
Can you add a comment on why it's needed to convert to a string?

I would prefer to use in `attached()`

```ts
    this.refSelectDAO.setAttribute("options", JSON.stringify(this.daoList));
```

Reply:
Removed `daoListStr`, as it is not needed starting v0.1.1 of `cl-webcomp-poc`plugin. Now passing the options directly as an object using the `init` method.